packages/sg-std/src/lib.rs

```diff
@@ -2,6 +2,8 @@ mod msg;
 mod route;
 
 pub const NATIVE_DENOM: &str = "ustars";
+// 3/1/2022 00:00:00GMT
+pub const GENESIS_MINT_START_TIME: u64 = 1646092800000000000;
```

Comment:
Do we want this to be a constant? Why not as part of the instantiation ?  Otherwise future mints will be stuck to this date (allowing front running)

Reply:
ooh nvm I just read the whole thing, thanks to github diff for being unhelpful

Reply:
on the other side should we check is not a time in the past (start_time> env.block_time) ?

Reply:
After genesis mint if the start_date is not set, people can mint right away. Don't want to bloat the contracts with too much logic either.

I'm not sure what the right approach is